### ISSUE TITLE
I am trying to do a project using matterport mask rcnn on colab. But wherever it starts training it utilizes cpu inspite of being connected to gpu. Where is the problem?
Tensorflow-1.5.0
Tensorflow-gpu-1.5.0
Keras-2.1.5

### ISSUE THREAD

**Issue by imtinan39** [matterport/Mask_RCNN#2104]
Tensorflow-1.5.0
Tensorflow-gpu-1.5.0
Keras-2.1.5

**Comment by ninthtide** [comment]
Same issue here.  Have read all articles I could find and tried multiple different versions of tensorflow-gpu and keras.

Graphics card shows all 8GB memory consumed (2070Super) but the actual GPU processor itself is ticking along at 0% - 2%; meanwhile CPU is showing a lot of active use

**Comment by imtinan39** [comment]
I have solved this issue by installing
Tensorflow==1.15.2
Tensorflow-gpu==1.15.2
Keras==2.1.5 
I have used colab

**Comment by ninthtide** [comment]
If it helps, it just might be a spectacular error on my part.  I was looking at my graphics card workload by using Windows task manager, Performance tab, then clicking the GPU section on the left hand side.  All the graphs looked like the card was operating at 0% load (although memory was fully consumed).

However I didn't immediately notice that the performance counters were selectable - i.e. very small drop-down arrows to the left of the graph title.  When I changed it to Cuda, I then saw my card was being used ~30% or so.... which looks like it's actually working correctly (bottleneck somewhere else I suppose).

Thanks for your guidance about the other package versions; I'll bear that in mind and will try them next time.

**Comment by burhr2** [comment]
Hi! kindly see this issue https://github.com/matterport/Mask_RCNN/issues/2312 there is a pointer to the same implementation that supports TensorFlow 2=>. If this solves your problem kindly close the issue so as others can navigate to other issues easier
